devices.js

```diff
@@ -138,6 +138,42 @@ const execute = (device, actions, callback, delay) => {
 };
 
 const devices = [
+    // Hampton Bay Fan Controller
+    {
+        zigbeeModel: ['HDC52EastwindFan'],
+        model: '99432',
+        vendor: 'Hampton Bay',
+        description: 'Universal wink enabled white ceiling fan premier remote control',
+        supports: 'on/off',
+        fromZigbee: [
+            fz.generic_state, fz.light_brightness, fz.ignore_fan_change, fz.generic_fan_mode, fz.ignore_onoff_change,
```

Comment:
The `generic_fan_mode` converter is missing, can you also add it?

Reply:
the device insert was from https://github.com/Koenkk/zigbee2mqtt/issues/1357

I'll reach out to the author for that code.